lib/command.h

```diff
@@ -399,6 +399,7 @@ struct cmd_node {
 #define IPV6_STR "IPv6 information\n"
 #define IP_ADDR_STR "IPv4 Address\n"
 #define IP6_ADDR_STR "IPv6 Address\n"
+#define BSR_STR "Bootstrap Router configuration\n"
```

Comment:
Why is this in the lib/command.h and not in pim_cmd.h?
If this is only going to be used by pim it should be defined there.